.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,46 @@
+<!--  Thanks for sending a pull request!  Here are some tips for you:
+
+1. If this is your first time, please read our contributor guidelines here:
+https://github.com/replicatedhq/kots/blob/master/CONTRIBUTING.md.
+2. Please label this pull request according to what type of issue you are addressing, especially if this is a release targeted pull request. For reference on required PR/issue labels, see here:
+https://github.com/replicatedhq/kots/labels?q=kind%2F
+3. Ensure you have added appropriate tests for your PR. For more information read here:
+https://github.com/replicatedhq/kots/blob/master/CONTRIBUTING.md#testing
+4. If the PR is unfinished, please mark it as a draft.
+-->
+
+#### What type of PR is this?
+
+<!--
+Add one of the following labels:
+/kind bug
+/kind documentation
+/kind enhancement
+/kind chore
+-->
+
+#### What this PR does / why we need it:
+
+#### Which issue(s) this PR fixes:
+<!--
+*Automatically closes linked issue when PR is merged.
+Usage: `Fixes #<issue number>`, or `Fixes (paste link of issue)`.
+-->
+Fixes #
+
+#### Special notes for your reviewer:
+
+#### Does this PR introduce a user-facing change?
+<!--
+If no, just write "NONE" in the release-note block below.
+If yes, a release note is required:
+-->
+```release-note
```

Comment:
Do we ask for a link to a PR into kots.io? This is a lot to ask of an external contributor.

Reply:
I don't think so.  Maybe we can ask that they supply a release note for the feature in the PR request and we can go through merged PRs and grab them when we release?  It be awesome if we had a way of automating this step.

Reply:
Agreed. Automating should be next steps